docs/hardware/devices/rak/peripherals.mdx

```diff
@@ -89,16 +90,39 @@ The RAK1910 is supported on the following base boards & slots:
 </TabItem>
 <TabItem value="Buzzer">
 
-## Buzzer
```

Comment:
This heading should remain, since it is the heading for the tab, and then there should be a new third level heading under it.

Reply:
@noon92 sorry I am being picky here but I'd really like us to make sure headings match tabs. Can you please update this to still be buzzer and add a new heading below for RAK? Thank you!